tests/system/Config/BaseConfigTest.php

```diff
@@ -53,6 +53,11 @@ protected function setUp(): void
         BaseConfig::setModules(new Modules()); // reset to clean copy of Modules
     }
 
+    protected function tearDown(): void
+    {
+        $this->resetServices();
```

Comment:
I prefer to have `parent::tearDown();` called first to ensure no BC break:

```suggestion
        parent::tearDown();
        $this->resetServices();
```